src/scss/style.scss

```diff
@@ -1024,57 +765,6 @@ input[type='number'] {
   margin-left: 5px;
 }
 
-.backend-status {
-  z-index: 3000;
-  display: grid;
-  position: fixed;
-  top: 0;
-  left: 0;
-  // background: #222;
-  // color: #bbb;
-  width: 100%;
-  grid-template: 20px / 16ch 1fr 1fr 25ch;
-  grid-template-areas: 'title status version jobs';
-  align-items: center;
-  font-size: small;
-}
-
-@media print {
-  .backend-status,
-  .random-potato,
-  .openSettings,
-  .print-controls,
-  footer {
-    display: none;
-  }
-  input,
-  textarea {
-    border-width: 0;
-  }
-}
-
-.bes-title {
-  justify-self: start;
-}
-
-.bes-status {
-  justify-self: start;
-  // color: lightgreen;
-}
-
-// .bes-error {
-//   color: red;
-// }
-
-.bes-version {
-  justify-self: end;
-}
-
-.bes-jobs {
-  justify-self: end;
-}
-.hidden {
-  position: absolute;
-  top: -1000px;
-  left: -1000px;
+.random-potato {
+  font-style: italic;
```

Comment:
I prefer the way the potato facts look without the italics, but I'll leave this up to you.

Reply:
Jack mentionned it in the refactor of the configurator it was `quirky`. Do we still keep those? Also we can't put it in the veil of the compilation.